basics.md

```diff
@@ -0,0 +1,108 @@
+Alright so you have installed the repository. Good job 🎉
+now let's began with writing some code-?
+
+**here's how to define a `variable`**
+Variable names can be arbitrarily long
+```
+
+```
+
+to **print** these, this is what you do
+```
+
+```
+
+you can't use variables with these names-
+```
+
+
+
+
+
+
+
+
+
+
+```
+
+**wanna add subtract multiply divide differentiate exponent or any mathematical funtion?**
+do it this way
+```
+
+
+
+
+
+
+```
+
+also you can define if you want to use BODMAS or PEMDAS using **modules** like this
```

Comment:
```suggestion
also you can define if you want to use BOMAS or PEMAS using **modules** like this
```
same reason as above

Reply:
alright wait I'll try and run some checks on divide and get bring out the changes soon - probably it's something with
```




```

Reply:
I agree. That's what I was thinking as well. Maybe if you `                              
` the ` ` it should work? I was reading on reddit somewhere that `  ` tends to `          ` so that could be it too.